distributor/storage.go

```diff
@@ -70,6 +70,7 @@ func (d *Distributor) readWithBackoff(ctx context.Context, ref torus.BlockRef, p
 		if err == nil {
 			return blk, err
 		}
+		clog.Errorf("failed peers, retry count %d: %v", i, err)
```

Comment:
this should be a warning not error, since we can still retry. we should log error only if all retried failed.